docs/source/guide-swarming.md

```diff
@@ -0,0 +1,10 @@
+## Swarming
+
+> Swarming is a process that automatically determines the best model for a given dataset. By "best", we mean the model that most accurately produces the desired output. Swarming figures out which optional components should go into a model (encoders, spatial pooler, temporal pooler, classifier, etc.), as well as the best parameter values to use for each component.
+
+We have plans to replace the current swarming library with a more universal parameter search library. This codebase is currently unmaintained. We currently have no API documentation for it.
```

Comment:
Do we have any public documents about it? I don't think we do. Do you think we should make the plan public?